src/ui/osx/TogglDesktop/test2/AppIconFactory.m

```diff
@@ -20,7 +20,13 @@ + (NSImage *)appIconWithType:(AppIconType)type
 			icon = [NSImage imageNamed:@"AppIconActive"];
 			break;
 		case AppIconTypeDefault :
-			icon = [NSImage imageNamed:@"AppIcon"];
+            if (@available(macOS 10.12.0, *))
+            {
+                icon = [NSImage imageNamed:@"AppIcon"];
+            } else {
+                icon = [NSImage imageNamed:NSImageNameApplicationIcon];
```

Comment:
For some reason, the icon app is blurry in OS 10.11

Add this safe code to be compatible.